### ISSUE TITLE
Mockery does not work when we generate and import mock in the same folder

### ISSUE THREAD

**Issue by dskarataev** [vektra/mockery#108]
I use mockery through go code generation tool:

```
//go:generate mockery -name IAnswerConverter -output mocks
```

and I have this error while generating the mock in the "converter" package because I import this mock in the "converter_test" package under the same directory:

```
question_converter_test.go:6:2: could not import sample_api/internal/handler/converter/mocks (cannot find package "sample_api/internal/handler/converter/mocks" in any of:
	/Users/user/go/src/ask_api/vendor/sample_api/internal/handler/converter/mocks (vendor tree)
	/Users/user/go/src/vendor/sample_api/internal/handler/converter/mocks
	/Users/user/.gobrew/versions/1.7.1/src/sample_api/internal/handler/converter/mocks (from $GOROOT)
	/Users/user/go/src/sample_api/internal/handler/converter/mocks (from $GOPATH))
Error parsing file:  couldn't load packages due to errors: /Users/user/go/src/sample_api/internal/handler/converter/answer_converter.go
question_converter_test.go:6:2: could not import sample_api/internal/handler/converter/mocks (cannot find package "sample_api/internal/handler/converter/mocks" in any of:
	/Users/user/go/src/ask_api/vendor/sample_api/internal/handler/converter/mocks (vendor tree)
	/Users/user/go/src/vendor/sample_api/internal/handler/converter/mocks
	/Users/user/.gobrew/versions/1.7.1/src/sample_api/internal/handler/converter/mocks (from $GOROOT)
	/Users/user/go/src/sample_api/internal/handler/converter/mocks (from $GOPATH))
Error parsing file:  couldn't load packages due to errors: /Users/user/go/src/sample_api/internal/handler/converter/question_converter.go
Unable to find IAnswerConverter in any go files under this path
internal/handler/converter/answer_converter.go:8: running "mockery": exit status 1
```

**Comment by dskarataev** [comment]
I think that PR #106 should fix the issue

**Comment by evanphx** [comment]
I merged #106, please reopen if this is still a problem.
